backend/src/controllers/UserController.js

```diff
@@ -0,0 +1,42 @@
+const connection = require('../database/connection');
+
+module.exports = {
+    async index(request, response) {
```

Comment:
Use two spaces to indent JavaScript code. This applies to this file in general not just this line.
eslint: [ident](https://eslint.org/docs/rules/indent)